### PR TITLE
Package creation for Network Threat Protection Essentials

### PR DESCRIPTION
**Pull request by v-sabiraj**
test doc- https://microsoftapc.sharepoint.com/:w:/r/teams/EcosystemCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B97B1F649-F1A4-4A26-8C38-52E8BFAC9148%7D&file=Sentinel%20Release%20Checklist%20-Network%20Threat%20Protection%20Essentials.docx&action=default&mobileredirect=true&share=IQFJ9rGXpPEmSow4Uui_rJFIAVbOzY_mk6WnWQsggSqwkYQ

### CODE REVIEW

**Comment by v-amolpatil** [comment]
@v-sabiraj Please check if we need to have skip validation as files are moved

**Comment by v-amolpatil** [comment]
@v-sabiraj Have you forgot to add testing document to this PR?

**Comment by v-amolpatil** [comment]
Add `status: Available` in Analytic Rule files

File: Solutions/Network Threat Protection Essentials/Data/Solution_NetworkThreatDetection.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...e.com/#create/azuresentinel.azure-sentinel-solution-amazonwebservicesazure-sentinel-solution-amazonwebservices)\r\n \r\n 3.[Microsoft Windows DNS](https://portal.azure.com/#create/azuresentinel.azure-sentinel-solution-dnsazure-sentinel-solution-dns)\r\n \r\n4.[Azure Firewall](https://ms.portal.azure.com/#create/sentinel4azurefirewall.sentinel4azurefirewallsentinel4azurefirewall)\r\n \r\n5.[Windows Forwarded Events](https://ms.portal.azure.com/#create/azuresentinel.azure-sentinel-solution-windowsforwardedeventsazure-sentinel-solution-windowsforwardedevents)\r\n \r\n6.[ZScaler Internet Access](https://ms.portal.azure.com/#create/zscaler1579058425289.zscaler_internet_access_msszia_msentinel_v1)\r\n \r\n7.[Palo Alto Networks](https://ms.portal.azure.com/#create/azuresentinel.azure-sentinel-solution-paloaltopanosazure-sentinel-solution-paloaltopanos)\r\n \r\n8.[Fortinet FortiGate](https://ms.portal.azure.com/#create/azuresentinel.azure-sentinel-solution-fortinetfortigateazure-sentinel-solution-fortinetfortigate)\r\n \r\n9.[Check Point](https://ms.portal.azure.com/#create/checkpoint.checkpoint-sentinel-solutionssentinel-1)\r\n \r\n**Keywords:** Malicious IP/User agent, DNS, TOR, mining",
```
Comment (v-amolpatil) [inline Solutions/Network Threat Protection Essentials/Data/Solution_NetworkThreatDetection.json:5]: remove `s` from `contains` to make the statement readable `The **Network Threat Protection Essentials** solution contain queries that identifies...`

**Comment by v-amolpatil** [comment]
Please remove `en-us` from below file(s):
image

**Comment by v-sabiraj** [comment]
@v-amolpatil, fixed all your comments.

**Comment by v-amolpatil** [comment]
@v-sabiraj  there is merge conflict

**Comment by v-amolpatil** [comment]
let us know once all changes are done

**Comment by v-amolpatil** [comment]
let us know once all changes are done

**Comment by v-amolpatil** [comment]
there is merged conflict

**Review by v-amolpatil — DISMISSED** [review]

**Review by v-amolpatil — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**